ChangeLog

```diff
@@ -13,6 +13,8 @@ Bugfix
    * Replace printf with mbedtls_printf in aria. Found by TrinityTonic in #1908.
    * Fix potential use-after-free in mbedtls_ssl_get_max_frag_len()
      and mbedtls_ssl_get_record_expansion() after a session reset. Fixes #1941.
+   * Return from various debugging routines immediately if the
```

Comment:
The debug routines are documented as taking an “SSL context” as argument, not an “SSL context or NULL”. This isn't a bug fix, it's a new feature, so it should be listed under “Changes”, or not listed at all in the changelog.

Reply:
Fixed in aa035d8.